openfisca_france/model/prelevements_obligatoires/impot_revenu/ir.py

```diff
@@ -699,43 +699,35 @@ def formula_2005_01_01(foyer_fiscal, period, parameters):
         maries_ou_pacses = foyer_fiscal('maries_ou_pacses', period)
         deficit_rcm = foyer_fiscal('deficit_rcm', period)
         f2ch = foyer_fiscal('f2ch', period)
+        f2da = foyer_fiscal('f2da', period)
         f2dc = foyer_fiscal('f2dc', period)
         f2ts = foyer_fiscal('f2ts', period)
         f2ca = foyer_fiscal('f2ca', period)
         f2fu = foyer_fiscal('f2fu', period)
         f2go = foyer_fiscal('f2go', period)
         f2gr = foyer_fiscal('f2gr', period)
         f2tr = foyer_fiscal('f2tr', period)
-        rvcm = parameters(period).impot_revenu.rvcm
-
-        # Add f2da to f2dc and f2ee to f2tr when no PFL
-        f2dc_bis = f2dc
-        f2tr_bis = f2tr
-        # # Calcul du revenu catégoriel
-        # 1.2 Revenus des valeurs et capitaux mobiliers
-        b12 = min_(f2ch, rvcm.abat_assvie * (1 + maries_ou_pacses))
-        TOT1 = f2ch - b12  # c12
-        # Part des frais s'imputant sur les revenus déclarés case DC
-        den = ((f2dc_bis + f2ts) != 0) * (f2dc_bis + f2ts) + ((f2dc_bis + f2ts) == 0)
-        F1 = f2ca / den * f2dc_bis  # f12
-        # Revenus de capitaux mobiliers nets de frais, ouvrant droit à abattement
-        # partie négative (à déduire des autres revenus nets de frais d'abattements
-        g12a = -min_(f2dc_bis * (1 - rvcm.taux_abattement_capitaux_mobiliers) - F1, 0)
-        # partie positive
-        g12b = max_(f2dc_bis * (1 - rvcm.taux_abattement_capitaux_mobiliers) - F1, 0)
-        rev = g12b + f2gr + f2fu * (1 - rvcm.taux_abattement_capitaux_mobiliers)
+        P = parameters(period).impot_revenu.rvcm
```

Comment:
Ne pas utiliser `P` mais un nom en minuscule: `rvcm_parameters` si tu ne veux pas utiliser `rvcm` ou un autre truc du genre.

Reply:
Ah ok, c'était justement pour harmoniser avec les notations utilisées dans les autres formules d'OF-France mais je change alors

Reply:
Ouais on veut virer les `P` c'est un reliquat de quand openfisca était en ... Matlab/Octave.